packages/jest-runtime/src/index.js

```diff
@@ -487,19 +544,29 @@ class Runtime {
     moduleRegistry: ModuleRegistry,
     from: Path,
   ) {
+    if (!this._environment) {
+      throw new Error(
+        `A module was required after the test suite ${this._path} finished.\n` +
+          `This usually means that you forgot to clean or mock an async operation`,
```

Comment:
nit: "In most cases this is because an async operation wasn't cleaned up or mocked properly."